### PR TITLE
Convert to Optimistic Goerli

### PR DESCRIPTION
**Pull request by emilyJLin95**
Modified scripts and README
Deleted node_modules, as that should not be committed

### CODE REVIEW

**Review by kevinbluer — APPROVED** [review]
Looks good from a quick review on my 📱

File: README.md [hunk 1]
```diff
@@ -114,19 +114,17 @@ You have several Optimistic Layer 2 networks to choose from, prepackaged in this
...
-    1) Acquire ETH for your Kovan wallet on MetaMask using a Kovan faucet.
-    2) Add Optimistic Ethereum as a Custom RPC to your Metamask wallet, using the [steps here](https://community.optimism.io/docs/developers/metamask.html#connecting-manually), except set the RPC URL to `https://optimism-kovan.infura.io/v3/" + <infuraKey>`
-    3) Visit [this website](https://gateway.optimism.io/) to bridge your Kovan ETH to Optimistic Kovan ETH
-    4) Ensure that your `optimistic_kovan` network in `truffle-config.ovm.js` is connected to your Optimistic Kovan wallet.
+- `optimistic_goerli`: Optimism has deployed a testnet to the Goerli network. The RPC endpoint is https://optimism-goerli.infura.io/v3/. In order to access this node for testing, you will need to connect a wallet (we suggest [MetaMask](https://metamask.io/)). Save your seed phrase in a `.env` file as `GOERLI_MNEMONIC`. Using an `.env` file for the mnemonic is safer practice because it is listed in `.gitignore` and thus will not be committed.
+  * Currently, we have the gasPrice for transactions on Optimistic Goerli set to zero. You should be able to use this network as configured at this time.
```
Comment (fainashalts) [inline README.md:118]: This is no longer true, as far as I know. Can you remove this bullet?

File: README.md [hunk 2]
```diff
@@ -168,14 +166,14 @@ Remember that there are some differences between the EVM and the OVM, and refer
...
 
 ### Communication Between Ethereum and Optimism Chains
 
-The information above should allow you to deploy to the Optimism Layer 2 chain. This is only the first step! Once you are ready to deploy your own contracts to function on Layer 1 using Layer 2, you will need to be aware of the [ways in which Layer 1 and Layer 2 interact in the Optimism ecosystem](http://community.optimism.io/docs/developers/integration.html#bridging-l1-and-l2). Keep an eye out for additional Truffle tooling and examples that elucidate this second step to full optimism integration!
+The information above should allow you to deploy to the Optimism Layer 2 chain. This is only the first step! Once you are ready to deploy your own contracts to function on Layer 1 using Layer 2, you will need to be aware of the [ways in which Layer 1 and Layer 2 interact in the Optimism ecosystem](http://community.optimism.io/docs/developers/integration.html#bridging-l1-and-l2). We have a [Optimism Bridge Box](https://trufflesuite.com/blog/introducing-the-optimism-bridge-truffle-box/?utm_source=github&utm_medium=devcommunity&utm_campaign=2022_Jul_optimism-box-readme_tutorial_content) that shows you just how to do that!
```
Comment (fainashalts) [inline README.md:176]:
```suggestion
The information above should allow you to deploy to the Optimism Layer 2 chain. This is only the first step! Once you are ready to deploy your own contracts to function on Layer 1 using Layer 2, you will need to be aware of the [ways in which Layer 1 and Layer 2 interact in the Optimism ecosystem](http://community.optimism.io/docs/developers/integration.html#bridging-l1-and-l2). We have an [Optimism Bridge Box](https://trufflesuite.com/blog/introducing-the-optimism-bridge-truffle-box/?utm_source=github&utm_medium=devcommunity&utm_campaign=2022_Jul_optimism-box-readme_tutorial_content) that shows you just how to do that!
```

**Review by fainashalts — CHANGES_REQUESTED** [review]
LGTM, with a couple tiny changes. Thanks for doing this!

**Pull request merged**